src/xdplwf/offloadrss.c

```diff
@@ -1262,7 +1262,8 @@ XdpLwfOffloadRssInspectOidRequest(
     *CompletionStatus = NDIS_STATUS_SUCCESS;
 
     if (Filter->Offload.Deactivated ||
-        Request->DATA.Oid != OID_GEN_RECEIVE_SCALE_PARAMETERS) {
+        Request->DATA.Oid != OID_GEN_RECEIVE_SCALE_PARAMETERS ||
+        Request->PortNumber != NDIS_DEFAULT_PORT_NUMBER) {
```

Comment:
Yeah. This is just a bugfix for now, since I realized XDP probably has undefined behavior as the default RSS config and the VF's RSS config both are treated the same by XDP. The bigger lift is to have defined behavior _and_ good performance.